pallets/runtime/tests/src/compliance_manager_test.rs

```diff
@@ -1228,9 +1065,9 @@ fn cm_test_case_11_we() {
         None
     ));
 
-    assert_valid_transfer!(ticker, owner_did, eve, 100);
+    assert_valid_transfer!(ticker, owner.did, eve, 100);
     let result =
-        ComplianceManager::verify_restriction_granular(&ticker, Some(owner_did), Some(eve));
+        ComplianceManager::verify_restriction_granular(&ticker, Some(owner.did), Some(eve));
```

Comment:
Modulo charlie/dave/eve, these occur thrice, so lets make a small closure out of this?